src/compiler/program.ts

```diff
@@ -3675,6 +3678,11 @@ namespace ts {
         return { diagnostics, sourceMaps: undefined, emittedFiles, emitSkipped: true };
     }
 
+    /*@internal*/
+    export function filterSemanticDiagnotics(diagnostic: readonly Diagnostic[], option: CompilerOptions): readonly Diagnostic[] {

```

Comment:
should probably be named `diagnostics`

Reply:
There's also a typo in the function name.